pkgs/tools/networking/openssh/default.nix

```diff
@@ -37,6 +37,8 @@ stdenv.mkDerivation rec {
 
   patches =
     [
+      ./fix-tunnel-forwarding-broken-in-7.7p1.patch
```

Comment:
I assume using `fetchpatch` was not possible for this package because it is in stdenv?

Reply:
ah vcuncat already used fetchpatch.